modules/community-algorithms/pages/k-means-clustering.adoc

```diff
@@ -0,0 +1,49 @@
+= k-Means Clustering
+
+In the k-Means Clustering algorithm, vertices are gradually sorted into an arbitrary number _k_ of clusters based on averaging their properties.
```

Comment:
After revisiting the algorithm yesterday, I think we need to call out the fact that your vertices need to already have embeddings as an attribute for k-means to work.